### PR TITLE
feat(javascript/rest-sveltekit): add SvelteKit example

### PR DESCRIPTION
**Pull request by baabouj**
add a fullstack example with [SvelteKit](https://kit.svelte.dev/) using [Svelte](https://svelte.dev/) and [Prisma Client](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client).

### CODE REVIEW

File: javascript/rest-sveltekit/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "rest-sveltekit",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "@fontsource/fira-mono": "^4.5.10",
+    "@neoconfetti/svelte": "^1.0.0",
```
Comment (ruheni) [inline javascript/rest-sveltekit/package.json]: Are these two dependencies used in the example?

Comment (baabouj) [reply javascript/rest-sveltekit/package.json]: No, they are not used actually 😅
They were used in the SvelteKit demo app when I created the project with `npm create svelte@latest`
Thanks for the notice @ruheni 👍😄

File: .github/readmes/javascript/_next-steps-rest-sveltekit.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
++  bio    String?
++  userId Int     @unique
++  user   User    @relation(fields: [userId], references: [id])
++}
+```
+
+Once you've updated your data model, you can execute the changes against your database with the following command:
+
+```
+npx prisma migrate dev
```
Comment (ruheni) [inline .github/readmes/javascript/_next-steps-rest-sveltekit.md]:
```suggestion
npx prisma migrate dev --name add-profile
```

**Comment by ruheni** [comment]
Hey 👋🏽

Thanks for working on this. Merging it now! 🚀

**Pull request merged**